sandstone/filewatcher.py

```diff
@@ -0,0 +1,43 @@
+from watchdog.observers import Observer
+import watchdog
+
+class Filewatcher(watchdog.events.FileSystemEventHandler):
+    """
+    Starts a watchdog instance to watch over the filesystem
+    Handles events - CREATE, REMOVE,MOVE
+    """
+    watched_directories = []
```

Comment:
`watched_directories` should be `_watched_directories` since they should be modified directly.